packages/d3fc-webgl/src/series/glBar.js

```diff
@@ -66,9 +66,9 @@ export default () => {
         program
             .buffers()
             .attribute('aXValue', xValueAttribute)
```

Comment:
How do you feel about the (in)consistency of when we use the value suffix?

Reply:
https://d3fc.io/api/series-api.html#ohlc

Reply:
Implemented.